trulens_eval/trulens_eval/tru.py

```diff
@@ -389,6 +390,44 @@ def Virtual(
 
         return TruVirtual(tru=self, app=app, **kwargs)
 
+    def create_benchmark_experiment(
+        self,
+        app_id: str,
+        feedback_fn: Callable,
+        agg_funcs: List[feedback.AggCallable],
+        benchmark_params: dict,
+        **kwargs: dict,
+    ) -> trulens_eval.tru_custom_app.TruCustomApp:
+        """Create a Custom app for special use case: benchmarking feedback functions.
+
+        Args:
+            app_id (str): user-defined identifier of the experiment run.
+            feedback_fn (Callable): feedback function of interest to perform meta-evaluation on.
+            agg_funcs (List[feedback.AggCallable]): list of aggregation functions to compute metrics for the benchmark.
+            benchmark_params (Any): parameters for the benchmarking experiment.
+
+        Returns:
+            trulens_eval.tru_custom_app.TruCustomApp: Custom app wrapper for benchmarking feedback functions.
+        """
+
+        from trulens_eval import TruCustomApp
+        from trulens_eval.feedback.benchmark_frameworks.tru_benchmark_experiment import (
+            TruBenchmarkExperiment,
+        )
+
```

Comment:
Is this a common python practice? To import inside a function ?

Reply:
I'm not sure how common it is but I've seen this in other code bases. This is done so here to avoid circular dependency, and I believe it's the same reason for several other occurrences in TruLens.